editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -5397,6 +5397,9 @@ void Node3DEditor::_update_gizmos_menu() {
 		const int plugin_state = gizmo_plugins_by_name[i]->get_state();
 		gizmos_menu->add_multistate_item(TTR(plugin_name), 3, plugin_state, i);
 		const int idx = gizmos_menu->get_item_index(i);
+		gizmos_menu->set_item_tooltip(
+				idx,
+				TTR("Click to toggle between visibility states.\n\nOpen eye: Gizmo is visible.\nClosed eye: Gizmo is hidden.\nPartially filled open eye: Gizmo is also visible through opaque surfaces (\"x-ray\")."));
```

Comment:
How about "Partially open eye" or "Half open eye" instead of "Partially filled open eye"? Seems a bit weird to mix the eye opening analogy with a color fill adjective. (And AFAIK the icon is meant to represent a partially open/half open eye.)